mysql/datadog_checks/mysql/util.py

```diff
@@ -57,3 +57,17 @@ def connect_with_autocommit(**connect_args):
         cursor.execute("SET AUTOCOMMIT=1")
 
     return db
+
+
+def get_list_chunks(lst, n):
+    """Yield successive n-sized chunks from lst."""
+    for i in range(0, len(lst), n):
+        yield lst[i : i + n]
+
+
+def fetch_and_convert_to_str(cursor):
```

Comment:
`None` - makes sense to convert.
Otherwise, I felt it was easier on the backend side to deal with strings.
The only thing I wanted to take care of were bools as otherwise there are these  yes, empty field = false etc...

Reply:
i don't strongly feel it's easier for the backend to deal with strings. as long as the data type is json serializable, it should not make much difference on the backend.

Reply:
AS discussed removed `fetch_and_convert_to_str`.